confmap/provider/aesprovider/provider.go

```diff
@@ -0,0 +1,108 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package aesprovider
```

Comment:
Run `make goporto` to automatically add a comment here. This is required to pass a CI check.